contrib/build-container.sh

```
@@ -229,4 +229,4 @@ BUILD_ARGS+=" --build-arg BUILD_TYPE=$BUILD_TYPE"
 
 show_build_options
 
-docker build --platform linux/$ARCH -f $DOCKER_FILE $BUILD_ARGS $TAG $NO_CACHE $BUILD_CONTEXT
+docker build --platform "linux/$ARCH" -f "$DOCKER_FILE" $BUILD_ARGS $TAG $NO_CACHE "$BUILD_CONTEXT"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)